bigtable/test/writeTest.php

```diff
@@ -46,7 +46,7 @@ public static function tearDownAfterClass(): void
 
     public function testWriteSimple()
     {
-        $output = $this->runSnippet('writes/write_simple', [
+        $output = $this->runSnippet('write_simple', [
```

Comment:
All of these should be changed to `runFunctionSnippet`
```suggestion
        $output = $this->runFunctionSnippet('write_simple', [
```